test/cbmc/proofs/make_common_makefile.py

```diff
@@ -211,13 +211,23 @@ def write_cbmcbatchyaml_target(opsys, _makefile):
     if opsys != "windows":
         _makefile.write(target)
 
+def write_solver(opsys, _makefile):
+    conditional = """
+ifneq ($(strip $(EXTERNAL_SAT_SOLVER)),)
+   SOLVER ?= --external-sat-solver $(EXTERNAL_SAT_SOLVER)
```

Comment:
Just as a matter of style: `SOLVER` is never meant to be set by the user (they're supposed to set `EXTERNAL_SAT_SOLVER`). So it's better to define the variable with `=` or `:=` as a signal that the Makefile will ignore any previously-set value.

Reply:
Done.